packages/task/src/browser/quick-open-task.ts

```diff
@@ -160,15 +160,16 @@ export class QuickOpenTask implements QuickOpenModel, QuickOpenHandler {
         this.items = [];
         this.actionProvider = undefined;
 
+        const configuredTasks = this.taskService.getConfiguredTasks();
         const providedTasks = await this.taskService.getProvidedTasks();
-        if (!providedTasks.length) {
+        if (!configuredTasks && !providedTasks.length) {
             this.items.push(new QuickOpenItem({
                 label: 'No tasks found',
                 run: (_mode: QuickOpenMode): boolean => false
             }));
         }
 
-        providedTasks.forEach(task => {
+        [...configuredTasks, ...providedTasks].forEach(task => {
```

Comment:
I think about use case when at first user has some detected task, then user configure the task.
Will 'Configure Tasks' action display both tasks after that? 
So two tasks with the same label will be displayed.
Is it expected behavior?

Reply:
@RomanNikitenko what do you believe is the correct behavior?
The `configure tasks...` menu should display all configured tasks, and provided (detected tasks) which have yet to be configured?

This means if the provided task is configured, it'll no longer show in the menu as detected?

Reply:
vscode doesn't display both configurations for this case.
"Run Task" menu of Theia displays only configured configuration if both are present (configured and detected)

Reply:
Thanks! I'll wait for your PR #5313 to resolve before continuing.